frontend/src/components/SplashScreen.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="splashscreen">
+
+    <img src="/images/loadingCircleThing.gif" alt="Loading..." class="loading-image">
+    <p class=" text">Loading</p>
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: "SplashScreen",
+};
+</script>
+
+<style>
+ .splashscreen {
+   height: 60vh;
+   display: flex;
+   justify-content: center;
+   flex-direction: column;
+ }
+
+ .loading-image {
```

Comment:
When you're using chatgpt (which is obvious here) you need to reference your use.